docs/markdown/level1.md

```diff
@@ -24,28 +7,27 @@ Operator: Automate the management of the Bestie application
 
 
 <aside class="notes">
-	YAML --> refactored --> Go
-	we need database deployment, job, route etc
-
+	- resource definition files
```

Comment:
we call them manifests .. but resource definition files actually sounds better..